docs/sqlml.rst

```diff
@@ -68,99 +55,64 @@ Rikai extends Spark SQL with four more SQL statements:
         DROP MODEL model_name;
 
 
-Rikai uses URL schema to decide which Model Registry to be used to resolve a
-ML Model. Once one ML model is via ``CREATE MODEL``,
-it can be used in Spark SQL directly:
+Once a ML model is loaded via ``CREATE MODEL``, it can be used in Spark SQL:
```

Comment:
I think `load` is wrong. It should be `create` or `register`.

Reply:
Fixed.